plugin/bufferline.vim

```diff
@@ -25,6 +25,7 @@ function! bufferline#enable()
       augroup bufferline_update
          au!
          au BufNew                 * call bufferline#update()
+         au BufNewFile             * lua  require'bufferline.state'.update_names()
```

Comment:
Autocommand moved here as per [this comment](https://github.com/romgrk/barbar.nvim/pull/58#discussion_r527922456).

Reply:
Yes but actually I meant we should call `bufferline#update()` there, sorry if I was unclear

Reply:
I understood— the linked comment provides an explanation as to why I didn't before checking in with you first.

Reply:
Oooh yes I missed that. I see now. Well one refactor that would be easier than storing names could be to pass an `update_names: bool` parameter down the stack through `bufferline#update()`, but if this is working I'm ok with keeping this autocmd there. In that case just put it at the bottom of the group and prefix it with a comment explaining why it's there.

I'd rather not run `update_names` at every `get_updated_buffers` call because that one runs on every `render` call and it would be repeating work that doesn't need to be repeated.

Reply:
I added `update_names` as a parameter in order to make it look more uniform. That way if another `autocmd` needs to be written you can just do:

```vim
bufferline#update(v:true)
```